Util/Xamarin.Build.Download/source/Xamarin.Build.Download.Tests/Xamarin.Build.Download.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net6</TargetFramework>

```

Comment:
Hmm, didn't realize this worked, most people put:
```suggestion
    <TargetFramework>net6.0</TargetFramework>

```

Reply:
`net6` works, but you were right `net6.0` is preferred.